src/SME.SR.Application/Commands/Frequencia/AcompanhamentoFrequencia/ObterDadosRelatorioAcompanhamentoFrequenciaCommandHandler.cs

```diff
@@ -87,12 +90,11 @@ public ObterDadosRelatorioAcompanhamentoFrequenciaCommandHandler(IMediator media
                 var alunosForaBimestre = new List<string>();
                 alunosSelecionados.ForEach(a =>
```

Comment:
Helio, sugestão.
Aqui poderia substituir por um foreach simples "foreach (var item in alunosSelecionados)", com isso é possível utilizar o await e remover o result da linha 93.
Realmente qdo utilizado "ForEach(" pode ocorre erro de concorrência, por isso a necessidade de usar o Result sem await, mas dessa forma q eu mencionei dá boa.
Aí se for alterar pode verificar a execução e confirmar que dá certo.